### PR TITLE
Transit network

### LINKED ISSUE #8: Transit Network Object with read/write methods
Create a transit network object in GeoPandas (most-likely a [Partridge](https://github.com/remix/partridge) object).

With methods to read the St Paul network from the transit network standard.

And methods to write the St Paul network in the transit network standard and/or Cube .lin files.

### PR DESCRIPTION
**Pull request by chryssac**
this fixes #8 

I checked in a very basic first draft just so I we can start talking about it. Can you post that example for the partridge object?

### CODE REVIEW

File: network_wrangler/TransitNetwork.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        args:
+        path: the path were the output files will be saved
+        filename: the name prefix of the transit files that are generated
+        '''
+        
+        if not os.path.exists(path):
+            WranglerLogger.debug("\nPath [%s] doesn't exist; creating." % path)
+            os.mkdir(path)
+            
+        else:
```
[the remark this answers is not in the document]
Comment (chryssac) [reply network_wrangler/TransitNetwork.py]: removed it

File: network_wrangler/TransitNetwork.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                transfers_file: {'service_id': None},
+                trips_file: {'service_id': None},
+                }
+        
+        
+        if not os.path.exists(path):
+            WranglerLogger.debug("\nPath [%s] doesn't exist; creating." % path)
+            os.mkdir(path)
+            
+        #TODO: do we want load_feed or load_geo_feed?
```
[the remark this answers is not in the document]
Comment (chryssac) [reply network_wrangler/TransitNetwork.py]: I went with the geo_config. Take a look at code for further comments

File: network_wrangler/TransitNetwork.py [hunk 3]
```diff
@@ -47,35 +48,24 @@ def read(self, agency_file: str, frequencies_file: str, routes_file: str,
...
             os.mkdir(path)
             
-        #TODO: do we want load_feed or load_geo_feed?
-        feed = ptg.load_feed(path, view)
+        config = geo_config()
+        #using load_feed so we can feed a config to it
+        feed = ptg.load_feed(path, view = None, config = config)
+        
         
+        #TODO: the above seems to be reading trips, routes, frequencies and agencies. Shapes, stops and transfers are not read.
```
Comment (e-lo) [inline network_wrangler/TransitNetwork.py]: its failing on shapes b/c the headers don't match what it is looking for in: https://github.com/remix/partridge/blob/0ba80fa30035e5e09fd8d7a7bdf1f28b93d53d03/partridge/config.py#L251

Comment (chryssac) [reply network_wrangler/TransitNetwork.py]: do we have to fix our input file? Who does that? Sijia?

**Pull request merged**